### PR TITLE
ci: cargo near new integration test + gh workflow to autorenew image tag/digest

### PR DESCRIPTION
**Pull request by dj8yfo**

### CODE REVIEW

File: .github/workflows/crontab_new_template_renewal.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          );
+
+          $mod_content | save -f cargo-near/src/commands/new/new-project-template/Cargo.toml.template
+
+          git diff
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v7
+        with:
+          title: update `cargo near new` template `image` and `image_digest`
```
Comment (dj8yfo) [inline .github/workflows/crontab_new_template_renewal.yml:49]: example of pr created https://github.com/dj8yfo/gh_self_create_pr/pull/3

File: integration-tests/tests/cargo_near_new.rs [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    Ok(())
+}
+
+/// tests logic of `../../cargo-near/src/commands/new/new-project-template/tests/test_basics.rs`,
+/// on contract, which is built by [cargo_near_build::build] from project, generated by `cargo near new`
+///
+/// differs from [test_cargo_test_on_generated_project] in the aspect, that current test
+/// uses [cargo_near_build::build] logic from current *cargo-near* branch, whereas
+/// [test_cargo_test_on_generated_project] uses [cargo_near_build::build] logic from
+/// latest published [near_workspaces::compile_project]
```
Comment (dj8yfo) [inline integration-tests/tests/cargo_near_new.rs:108]: this test addresses suggestion in https://github.com/near/cargo-near/pull/229#issuecomment-2399227236

**Comment by dj8yfo** [comment]
@race-of-sloths include

**Review by frol — APPROVED** [review]
@race-of-sloths score 13

**Pull request merged**